core/message-tags-3.3.md

```diff
@@ -125,7 +128,7 @@ Servers MUST NOT add tag data exceeding 510 bytes to messages.
     <client_max>   (4096)  :: '@' <tag_data 4094> ' '
     <combined_max> (4607)  :: '@' <tag_data  510> ';' <tag_data 4094> ' '
 
-Servers MUST reply with the `ERR_INPUTTOOLONG` (`417`) error numeric if a client sends a message with more tag data than the allowed limit. Servers MUST NOT truncate tags to allow lines that exceed this limit.
+Servers MUST reply with the `ERR_INPUTTOOLONG` (`417`) error numeric if a client sends a message with more tag data than the allowed limit. Servers MUST NOT truncate tags but MUST always reject lines that exceed this limit.
```

Comment:
The wording of this sentence feels strange. The 'but' feels like it should take the sentence in a new direction, but really just tells servers to do the next step of the same thing.

Can we change 'but' to 'and' to make this? Feels easier to read and more clear on the first pass:
"Servers MUST NOT truncate tags and MUST always reject lines that exceed this limit"